Project/js/content.js

```diff
@@ -0,0 +1,85 @@
+const bikeAPI = new URL('https://api.citybik.es/v2/networks');
+const wikipediaAPI = new URL("https://simple.wikipedia.org/w/api.php?action=opensearch&origin=*");
+const locations = [];
+const limit = 10;
+
+fetch(bikeAPI)
+    .then(response => response.json())
+    .then(data => displayCities(data.networks))
+    .catch(error => console.error(error.message));
+
+function displayCities(data) {
+    const list = document.querySelector('.cities-list');
+    const frag = document.createDocumentFragment();
+
+    for (let i = 0; i < limit; i++) {
+        const currentLocation = {};
+
+        currentLocation.lat = Math.round(data[i].location.latitude * 1000) / 1000;
+        currentLocation.lng = Math.round(data[i].location.longitude * 1000) / 1000;
+        locations.push(currentLocation);
+        frag.appendChild(createItem(data, i));
+    }
+    list.appendChild(frag);
+    initMap(locations);
+}
+
+function initMap() {
+    const start = locations[0];
+    const map = new google.maps.Map(document.getElementById('map'), {
+        zoom: 4,
+        center: start
+    });
+
+    const labels = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+    for (let j = 0; j < locations.length; j++) {
+        new google.maps.Marker({
+            position: locations[j],
+            label: labels[j],
+            map: map
+        });
+    }
+}
+
+function createItem(data, index) {
```

Comment:
I don't see why you are passing all `data` and `index` instead of specific item from the table